app/src/main/java/com/duckduckgo/app/settings/SettingsViewModel.kt

```diff
@@ -44,6 +44,7 @@ class SettingsViewModel @Inject constructor(
         val loading: Boolean = true,
         val version: String = "",
         val lightThemeEnabled: Boolean = false,
+        val darkModeEnabled: Boolean = false,
         val autoCompleteSuggestionsEnabled: Boolean = true,
         val showDefaultBrowserSetting: Boolean = false,
         val isAppDefaultBrowser: Boolean = false,
```

Comment:
You don't need to specify the type.

Reply:
Disagree for multiple reasons.

1. Kotlin type inference puts stress on the IDE (a small amount though although noticeable on big files)
2. Consistency, the rest of the data class has the types specified
3. Most important one, improves readability and serves as documentation (specially on more complex functions)